components/wrong-network-modal.tsx

```diff
@@ -51,8 +51,8 @@ export default function WrongNetworkModal({
           },
         ],
       });
-    } catch (error: any) {
-      if (error.code === 4902) {
+    } catch (error) {
```

Comment:
you can type _this_ as `as {code?: number}` (or something that works) and then you don't have to repeat the typing

Reply:
this was the only way to get type, I created a type const to improve this repeatability